### DISCUSSION TITLE
Are we planning to integrate timepicker?

### DISCUSSION THREAD

**Discussion by janjanarnaldo** [gpbl/react-day-picker#1414] [Ideas]
**Is your feature request related to a problem? Please describe.**
My project is currently invested heavily in this calendar picker with regards to using it into Form and custom conditions. My problem is that I am using a separate library for TimePicker.

It was all good and fine until my UX designed a form with date and time picker into 1 input field, essentially merging the 2 form components together and making it work as one.

**Describe the solution you'd like**
My question is, is there a future plan or support for time picker in this project?

**Describe alternatives you've considered**
Planning to create a component with these 2 components merged together but it will need a lot of hacking and stuff just to make it work as one. Not to mention I'm using react-hook-form for my forms with makes it hard to register 2 components to be 1.

**Additional context**
Remove everything and use another library with day and time picker options, but risk refactoring all the existing conditions we have using this component.

**Comment by gpbl** [comment]
@janjanarnaldo there're no plans for a "time picker". I'm curious however about the hacking you need to integrate an existing one. Could you provide an example? Not sure also how a "time picker" should look like.

Would something similar work?

https://codesandbox.io/s/sandpack-project-forked-3rn8qb?file=/App.tsx

**Reply by janjanarnaldo** [reply]
Yes mate definitely something like that but I wrapped DayPicker and TimePicker components in OverlayComponent so I could put them side by side. Most of the handling is the same with your sandbox.

![image](https://user-images.githubusercontent.com/13392638/160306909-d7708212-c281-41b6-9576-d2ce6dc800bb.png)

**Reply by gpbl** [reply]
@janjanarnaldo looks neat! You could open source it. No plans to add something similar.... Actually, yours is the intended use of DayPicker: just the calendar 👌
